luda-editor/luda-editor-client/src/app/editor/mod.rs

```diff
@@ -169,29 +193,30 @@ impl namui::Entity for Editor {
                 NamuiEvent::MouseUp(global_xy) => {
                     let job = self.job.clone();
                     match job {
+                        // TODO : Make these simple using trait
                         Some(Job::MoveCameraClip(mut job)) => {
                             job.last_global_mouse_xy = *global_xy;
-                            job.execute(&mut self.timeline);
+                            job.execute(self);
```

Comment:
During work for this PR, I changed that job update `Editor`, not `Timeline`. I needed this changes to move sequence data from timeline to editor. most of these jobs edit sequence. 
I will make it sure next time with `Job` trait someday.